pkg/commands/publisher_test.go

```diff
@@ -72,6 +73,16 @@ func TestPublishImages(t *testing.T) {
 		if err != nil {
 			t.Fatalf("%s: MakeBuilder(): %v", test.description, err)
 		}
+
+		defaultBaseImageRef, err := build.DefaultBaseImage()
```

Comment:
This doesn't seem to be used later except just to check that it's set? Can we test that it's a specific expected value?

Reply:
yep absolutely!